paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -51,12 +51,13 @@ inline MKLDNNMemoryFormat GetWeightsFormat(const MKLDNNMemoryFormat format,
   }
 }
 
-static mkldnn::memory::data_type GetDstType(bool is_int8,
+static mkldnn::memory::data_type GetDstType(bool is_int8, bool is_bfloat16,
                                             bool force_fp32_output,
                                             std::string fuse_activation,
                                             bool fuse_residual_conn,
                                             const Tensor* residual_param) {
-  auto dst_dt = mkldnn::memory::data_type::f32;  // uint8_t, int8_t, float
+  auto dst_dt =
+      mkldnn::memory::data_type::f32;  // uint8_t, int8_t, float, bfloat16
```

Comment:
I think this comment could be removed completely.

Reply:
Done